### ISSUE TITLE
CI: QEMU cluster not starting up, ssh timeout

### ISSUE THREAD

**Issue by pohly** [intel/pmem-csi#518]
This happened in a variety of recent job runs, for example:
https://cloudnative-k8sci.southcentralus.cloudapp.azure.com/view/pmem-csi/job/pmem-csi/job/devel/136/execution/node/94/log/

```
11:07:49  ERROR: timeout accessing 172.17.0.4 through ssh
11:07:49  Cluster creation failed.
11:07:49  govm status:
11:07:49  ID         Name                  Namespace IP      
11:07:49  49a0dfccb7 pmem-csi-govm-worker3 jenkins           
11:07:49  0a326178db pmem-csi-govm-worker2 jenkins           
11:07:49  40e2fe15ed pmem-csi-govm-worker1 jenkins           
11:07:49  afa23b4d78 pmem-csi-govm-master  jenkins           
11:07:49  Docker status:
11:07:49  CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS                            PORTS                    NAMES
11:07:49  49a0dfccb7bd        govm/govm:latest    "/usr/local/bin/star…"   2 minutes ago       Restarting (139) 43 seconds ago                            govm.jenkins.pmem-csi-govm-worker3
11:07:49  0a326178db67        govm/govm:latest    "/usr/local/bin/star…"   2 minutes ago       Restarting (139) 47 seconds ago                            govm.jenkins.pmem-csi-govm-worker2
11:07:49  40e2fe15edba        govm/govm:latest    "/usr/local/bin/star…"   2 minutes ago       Restarting (139) 47 seconds ago                            govm.jenkins.pmem-csi-govm-worker1
11:07:49  afa23b4d78c1        govm/govm:latest    "/usr/local/bin/star…"   2 minutes ago       Restarting (139) 51 seconds ago                            govm.jenkins.pmem-csi-govm-master
11:07:49  5ead7ca4f4ae        registry:2          "/entrypoint.sh /etc…"   12 minutes ago      Up 12 minutes                     0.0.0.0:5000->5000/tcp   registry
11:07:49  8454dabf0b70        1c0fc60c7ade        "sleep infinity"         13 minutes ago      Up 13 minutes                                              builder
```

Unfortunately that debug output does not include docker logs, so it is unclear *why* the container was restarting. Will add that...

**Comment by pohly** [comment]
Will add that...
> See https://github.com/intel/pmem-csi/pull/519

**Comment by pohly** [comment]
Resolved by PR #519
